Makefile

```diff
@@ -266,26 +266,25 @@ jstest: build-addons build-addons-napi ## Runs addon tests and JS tests
 .PHONY: test
 # This does not run tests of third-party libraries inside deps.
 test: all ## Runs default tests, linters, and builds docs.
-	# Build the addons before running the tests so the test results
-	# can be displayed together
+	@echo "Build the addons before running the tests so the test results"
+	@echo "can be displayed together"
 	$(MAKE) -s build-addons
 	$(MAKE) -s build-addons-napi
 	$(MAKE) -s cctest
 	$(MAKE) -s jstest
 
 .PHONY: test-only
 test-only: all  ## For a quick test, does not run linter or build docs.
-	# Build the addons before running the tests so the test results
-	# can be displayed together
-	$(MAKE) build-addons
```

Comment:
Remove on purpose?

Reply:
No, that was a mistake 😞  Thanks for catching that. I'll update shortly.